client/client.go

```diff
@@ -134,9 +95,12 @@ func (c *Client) createConsumerChannel() {
 func New(url string, args ...interface{}) *Client {
 	c := &Client{
 		dialconfig: amqp.Config{
-			Dial: connection.DefaultDialer,
+			// Dial: connection.DefaultDialer,
```

Comment:
Do we break the fix for race conditions by using `DefaultDialer` (which is currently commented out)? If so, do you know why?

Reply:
I'll enable it again to see what happens, but I don't think it will matter.